src/main/java/seedu/address/model/item/Item.java

```diff
@@ -9,13 +9,15 @@
 import java.util.Objects;
 import java.util.Set;
 
+import seedu.address.model.Inventory;
 import seedu.address.model.tag.Tag;
+import seedu.address.ui.View;
 
 /**
  * Represents an Item in the Inventoryinator.
  * Guarantees: details are present and not null, field values are validated, immutable.
  */
-public class Item {
+public class Item extends Inventory {
```

Comment:
May not be in line with good OOP practices. Consider alternative representation.